File: library/src/main/java/com/proxerme/library/enums/SortType.java
package com.proxerme.library.enums;

import com.squareup.moshi.Json;

/**
 * Enum holding the available sort types.
 *
 * @author Ruben Gees
 */
public enum SortType {
    @Json(name = "ASC")ASCENDING,
    @Json(name = "DESC")DESCENDING
}

File: library/src/main/kotlin/me/proxer/library/enums/SortType.kt
package me.proxer.library.enums

import com.squareup.moshi.Json
import com.squareup.moshi.JsonClass

/**
 * Enum holding the available sort types.
 *
 * @author Ruben Gees
 */
@JsonClass(generateAdapter = false)
enum class SortType {

    @Json(name = "ASC")
    ASCENDING,

    @Json(name = "DESC")
    DESCENDING
}
